server.js

```diff
@@ -1,22 +1,9 @@
-const environment = process.env.NODE_ENV || 'development';
-const configuration = require('./knexfile')[environment];
-const database = require('knex')(configuration);
-const express = require('express');
-const bodyParser = require('body-parser')
-const app = express();
-app.use(bodyParser.json());
-app.set('port', process.env.PORT || 3001);
+const app = require('./app')
```

Comment:
Glad we broke out our server code.  This will be much cleaner and we shouldn't run into any of the port issues we were having.